CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+# [0.1.0](https://github.com/SwissDataScienceCenter/amalthea/compare/da735ced323eacb38fd010e4ae0a0479fb2bf310...0.1.0) (2021-09-15)
+
+
+### Bug Fixes
```

Comment:
Thanks for adding the CHANGELOG file. Since this is the first entry here, I think a very short list of the most important features which are already included is more meaningful than the conventional changelog.